client/reader/sidebar/reader-sidebar-nudges/index.jsx

```diff
@@ -67,11 +70,12 @@ function mapStateToProps( state ) {
 		siteSlug,
 		isEligibleForFreeToPaidUpsellNudge:
 			siteCount === 1 && // available when a user owns one site only
-			'en' === getLocaleSlug( state ) && // only for English speakers
-			'US' === countryCode && // only for US residents
-			! isDomainOnlySite( state, siteId ) && // not for domain only sites
 			! isJetpackSite( state, siteId ) && // not for Jetpack sites
-			isEligibleForFreeToPaidUpsell( state, siteId ),
+			! isDomainOnlySite( state, siteId ) && // not for domain only sites
+			isEligibleForFreeToPaidUpsell( state, siteId ) &&
+			// This test is not for English speaking US residents.
+			( ( 'en' === userLocale && 'US' === countryCode ) ||
```

Comment:
We need to include all `en` variations, since someone with en-gb in the US will be assigned to the test.

Reply:
> We need to include all en variations, since someone with en-gb in the US will be assigned to the test.

@niranjan-uma-shankar The nudge should show up to all eligible English US users. English speakers outside of US would be included in the a/b test. That line of code ensures EN US users to not be assigned to the test.

Reply:
Sorry if I'm misunderstanding this, but just to clarify, wouldn't this also assign EN US users to the test if they have their language preference to EN-GB? I was wondering if we should exclude such users. I'm thinking on the lines of [using this config property](https://github.com/Automattic/wp-calypso/blob/77318c9b32568459aeb80639f8e01a137b3a5ee1/config/_shared.json#L119), like how it's used [here](https://github.com/Automattic/wp-calypso/blob/de18db59f34af488ff47092e28b596ad9dade68a/client/me/concierge/book/info-step.js#L132). What do you think?

```
( includes( config( 'english_locales' ), userLocale ) && 'US' === countryCode ) ||

```

Reply:
> wouldn't this also assign EN US users to the test if they have their language preference to EN-GB?

@niranjan-uma-shankar You're right. But I think it would be okay since this PR is designed to test against all other users who are not the previous audience.

Reply:
Okay 👍